RELEASING/Dockerfile.from_local_tarball

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM python:3.7-jessie
+FROM python:3.7-buster
```

Comment:
`python:3.7-jessie` does not exist anymore, so I wonder if `python:3.7-buster` will be removed also